tag.go

```diff
@@ -263,6 +266,10 @@ func (t *Tag) SetVersion(version byte) {
 // Save writes t to the file. If there are no frames in tag, Save will write
 // only music part without any ID3v2 information.
 func (t *Tag) Save() error {
+	if t.file == nil {
+		return fmt.Errorf("Parser not inited with file, it's just a stream")
```

Comment:
I think, we should create special error for it and return it:
```go
// ErrNoFile is returned in Save and Close method, if tag was not
// initialized with file.
var ErrNoFile = errors.New('tag was not initialized with file')
```
And then write in docs of `Save` and `Close` this line:
```
// If tag was initiliazed not with file but from ParseReader,
// it returns ErrNoFile.
```